Dockerfile

```diff
@@ -38,6 +38,9 @@ RUN --mount=type=cache,target=/root/.cache/pip \
 # copy input files
 COPY csrc csrc
 COPY setup.py setup.py
+COPY hipify.py hipify.py
```

Comment:
I think the `hipify.py` and `utils.cmake` files should go into a `cmake` folder to avoid having too many files in the root directory.